# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1435861398 14400
#      Thu Jul 02 14:23:18 2015 -0400
# Node ID 64f6d054362615c70495f99dd999d4e38ce71432
# Parent  b6a59cc96bfa22cb92dc63a0b3cdb3f726dbe3ed
eliminate unused static functions

* Canvas.cc (rotate_enabled, zoom_direction): Delete.

diff --git a/libgui/graphics/Canvas.cc b/libgui/graphics/Canvas.cc
--- a/libgui/graphics/Canvas.cc
+++ b/libgui/graphics/Canvas.cc
@@ -343,28 +343,16 @@ pan_mode (const graphics_object figObj)
     = Utils::properties<figure> (figObj).get___pan_mode__ ();
 
   octave_scalar_map pm = ov_pm.scalar_map_value ();
 
   return pm.contents ("Motion").string_value ();
 }
 
 static bool
-rotate_enabled (const graphics_object figObj)
-{
-  // Getting rotate mode property:
-  octave_value ov_rm
-    = Utils::properties<figure> (figObj).get___rotate_mode__ ();
-
-  octave_scalar_map rm = ov_rm.scalar_map_value ();
-
-  return rm.contents ("Enable").string_value () == "on";
-}
-
-static bool
 zoom_enabled (const graphics_object figObj)
 {
   // Getting zoom mode property:
   octave_value ov_zm
     = Utils::properties<figure> (figObj).get___zoom_mode__ ();
 
   octave_scalar_map zm = ov_zm.scalar_map_value ();
 
@@ -378,28 +366,16 @@ zoom_mode (const graphics_object figObj)
   octave_value ov_zm
     = Utils::properties<figure> (figObj).get___zoom_mode__ ();
 
   octave_scalar_map zm = ov_zm.scalar_map_value ();
 
   return zm.contents ("Motion").string_value ();
 }
 
-static std::string
-zoom_direction (const graphics_object figObj)
-{
-  // Getting zoom mode property:
-  octave_value ov_zm
-    = Utils::properties<figure> (figObj).get___zoom_mode__ ();
-
-  octave_scalar_map zm = ov_zm.scalar_map_value ();
-
-  return zm.contents ("Direction").string_value ();
-}
-
 void 
 Canvas::select_object (graphics_object obj, QMouseEvent* event, 
                        graphics_object &currentObj, graphics_object &axesObj)
 {
   QList<graphics_object> axesList;
   Matrix children = obj.get_properties ().get_all_children ();
   octave_idx_type num_children = children.numel ();
 
